### ISSUE TITLE
Asset licenses

### ISSUE THREAD

**Issue by Rampoina** [00-Evan/shattered-pixel-dungeon#380]
What license are the assets licensed under?
I've seen that Pixel Dungeon has a similar issue that hasn't been answered in 5 years https://github.com/watabou/pixel-dungeon/issues/58 but maybe you can help clarify that.

**Comment by 00-Evan** [comment]
Afraid I'm not an authority on what license PD (and by extension Shattered) uses for assets. GPLv3 is a safe assumption though, given that's the license which is used for the code, and the copy of the license is in the root directory, which contains the assets.

**Comment by Rampoina** [comment]
> Afraid I'm not an authority on what license PD (and by extension Shattered) uses for assets. 

Hello, thanks for your response. I thought that would be the case.

> GPLv3 is a safe assumption though, given that's the license which is used for the code, and the copy of the license is in the root directory, which contains the assets.

I don't think so, there's many projects that have the code as GPL (or other free software licenses) but don't specify the asset licenses, and then when you ask it's copyrighted. The copy of the code license is almost always in the root directory. By default unlicensed files are copyrighted too under most jurisdictions (as far as I know)

**Comment by 00-Evan** [comment]
In that case then every single PD mod would be violating Watabou's copyright, and you'll notice he hasn't taken action against any of them, and infact has cooperated with mod developers in the past. Again, obviously I cannot give a certain answer, but I do believe it's a safe assumption.

**Comment by Rampoina** [comment]
I'll contact watabou. Thank you.

**Comment by pazkero** [comment]
> By default unlicensed files are copyrighted too under most jurisdictions (as far as I know)

Most jurisdictions just declare "restrictive interpretation to attain the license objectives" in some way or another. For GPL that usually will not cover assets, but this depends strictly on court understanding. However, if the only file provided is the GPL license with no EULA then "entirety of agreement" might apply, and because Shattered is a fork as defined by the GPL itself, under most jurisdictions, eventual complains in that regard will be treated in an ex nunc basis (so from notification of error onwards).

In other words, unlicensed files on a GPL project should be treated as the most restrictive interpretation the general license allows, which will likely be GPL for purposes of derivative works and copyright for everything else.

**Comment by Pr0methean** [comment]
@pazkero You seem to know more about the law than anyone else in this thread. Can you recommend a lawyer who does pro-bono work to benefit open-source software projects such as this?

**Comment by Pr0methean** [comment]
Or maybe @bucchere knows someone, since he's worked with Lawrence Lessig.

**Comment by bucchere** [comment]
Have you checked out https://softwarefreedom.org? (Not an endorsement as I've never worked with them, so YMMV.)

**Comment by pazkero** [comment]
I was going to endorse SFLC as well 😅

They are probably the most famous, but I also never worked personally with them.
